### PR TITLE
Docs: adding bracket sections to framework guides; updating links.

### PR DESCRIPTION
**Pull request by godai78**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/guides/contributing/git-commit-message-convention.html))

Docs: adding the recap and what's next sections around the frameworks integration guides. Closes cksource/ckeditor5-internal#1444

---

### Additional information

_For example – encountered issues, assumptions you had to make, other affected tickets, etc._

### CODE REVIEW

File: docs/installation/getting-started/basic-api.md [hunk 1]
```diff
@@ -217,5 +217,7 @@ ClassicEditor.create( ... ); // [Function]
 
 Having read this guide, you know how to communicate with the editor, but remember that CKEditor 5 offers a rich API to interact with it. Check out the {@link api/index API documentation} for more.
 
-You have almost completed the CKEditor 5's quick start path for beginners, well done! It is important to know how you can maintain your editor, so take a look at the {@link installation/getting-started/maintenance last article from this section} to learn more.
+If you would like to integrate you CKEditor 5 installation with one of the popular Java Script frameworks, {@link installation/frameworks/overview we have a dedicated guide for that}.
```
Comment (Mgsy) [inline docs/installation/getting-started/basic-api.md:220]: `Java Script` -> `JavaScript`

**Review by dufipl — APPROVED** [review]
Updated outdated link in the Frameworks CSS guide. Besides that LGTM 👍

File: docs/installation/frameworks/overview.md [hunk 2]
```diff
@@ -53,3 +59,15 @@ ckeditor {
...
+## Compatibility with CSS frameworks
+
+CKEditor 5 is compatible with popular CSS frameworks, such as [Bootstrap](https://getbootstrap.com/) or [Foundation](https://get.foundation/). Such integrations, however, often require additional changes and adjustments {@link installation/frameworks/css gathered in this guide}.
+
+<info-box hint>
+**What's next?**
+
+After reading this guide, you know how to integrate the editor with your own system. This step almost concludes the beginner's path with getting to know CKEditor 5.
+
+If you haven't done so yet, now now is a great time take a look at the {@link installation/getting-started/maintenance last article from this section} to learn more on how you can maintain your editor.
```
[the remark this answers is not in the document]
Comment (godai78) [reply docs/installation/frameworks/overview.md:72]: Fixed in commit.

File: docs/installation/frameworks/overview.md [hunk 3]
```diff
@@ -6,11 +6,17 @@ menu-title: Overview
 
 # Integrating CKEditor 5 with JavaScript frameworks
 
+<info-box hint>
+**Quick recap**
+
+In the previous guides you have learned several different methods of getting you CKEditor 5 installation up and running. Now you will learn how to utilize the power of Java Script framework to include your editor as a component.
```
Comment (Mgsy) [inline docs/installation/frameworks/overview.md:12]: `Java Script` -> `JavaScript`

Also, this sentence sounds a bit weird to me:

> Now you will learn how to utilize the power of Java Script framework to include your editor as a component

I think I understand what you mean here, but maybe we can rephrase it and emphasise that we have ready-to-use CKEditor 5 components for Angular, React and Vue?

Comment (godai78) [reply docs/installation/frameworks/overview.md:12]: Fixed in commit.

File: docs/installation/getting-started/basic-api.md (hunk 1 above)
Comment (godai78) [reply docs/installation/getting-started/basic-api.md]: Fixed in commit.

**Pull request merged**